### ISSUE TITLE
Close Tab History in context menu (FX57)

### ISSUE THREAD

**Issue by MrGlasspoole** [codefisher/toolbar_buttons#5]
Hi,
is it possible to at the tab history also to the context menu if you make a right click on a tab?

**Comment by codefisher** [comment]
Yes, already done this in my other extension, undo close tab.  I will copy it across for the next update.

**Comment by MrGlasspoole** [comment]
In undo close tab? I mean that is already there in Firefox without that Add-on?

**Comment by codefisher** [comment]
https://addons.mozilla.org/en-US/firefox/addon/undo-close-tab-button/?src=ss

**Comment by MrGlasspoole** [comment]
Ah OK it also has a history.
So what is the difference between "Undo Close Tab Button" and "Close Tab History Button"?
I don't get it...

**Comment by codefisher** [comment]
Undo Close Tab has two modes of operation, some wanted both, so I ended up making two extensions. Since with how WebExtensions work I could not put both in the one extension.
